docs/user-guide/styling-your-docs.md

```diff
@@ -30,12 +30,11 @@ the [custom themes](#custom-themes) section below.
 
 ![ReadTheDocs](https://docs.readthedocs.org/en/latest/_images/screen_mobile.png)
 
-## Bootstrap and Bootswatch Themes
+## Bootstrap and Bootswatch themes
```

Comment:
This is a section header/subtitle so it should follow the [capitalization rules][1] of a title. In other words, "Themes" should be capitalized.

[1]: http://capitalizemytitle.com/title-capitalization-rules/